cachi2/core/package_managers/bundler/scripts/lockfile_parser.rb

```diff
@@ -11,7 +11,9 @@
 lockfile_parser.specs.each do |spec|
     parsed_spec = {
       name: spec.name,
-      version: spec.version.to_s
+      version: spec.version.to_s,
+      full_name: spec.full_name,
```

Comment:
I don't think we need `full_name`, it can be easily constructed from `name` and `version` as you have in the code.
That said, I wonder if we can move the platform attribute to the `GemDependency` class and make it default to ruby (for example). I see a lot of duplication in both classes.

+ When I look at ~our favorite~ PURL spec [doc](https://github.com/package-url/purl-spec/blob/master/PURL-TYPES.rst#gem) I see a platform qualifier. The default is `ruby`.